apps/settings/js/panels/call/call_forwarding.js

```diff
@@ -0,0 +1,225 @@
+/**
+ * Make sure all operations would be executed one by one.
+ *
+ * @module call/call_forwarding
+ */
+define(function(require) {
```

Comment:
The module should provide:
- A method for refreshing call forwarding settings (sync with the carrier)
- A method for setting call forwarding settings
- Observable properties presenting the current call forwarding settings.
- Observable property reporting the current querying state (i.e. requesting).

Note that we should have a dedicated call forwarding instance for each sim card.